en/advanced_config/land_detector.md

```diff
@@ -55,7 +55,7 @@ This state is reached if following conditions are true for 0.35 seconds:
 
 - no vertical movement ([LNDMC_Z_VEL_MAX](../advanced_config/parameter_reference.md#LNDMC_Z_VEL_MAX))
 - no horizontal movement ([LNDMC_XY_VEL_MAX](../advanced_config/parameter_reference.md#LNDMC_XY_VEL_MAX))
-- low thrust `MPC_THR_MIN + (MPC_THR_HOVER - MPC_THR_MIN) * 0.3` or velocity setpoint is 0.9 of land speed but vehicle has no vertical movement.
+- lower thrust then [MPC_THR_MIN](../advanced_config/parameter_reference.md#MPC_THR_MIN) + ([MPC_THR_HOVER](../advanced_config/parameter_reference.md#MPC_THR_HOVER) - [MPC_THR_MIN](../advanced_config/parameter_reference.md#MPC_THR_MIN)) * [LNDMC_LOW_T_THR](../advanced_config/parameter_reference.md#LNDMC_LOW_T_THR) or velocity setpoint is 0.9 of land speed but vehicle has no vertical movement.
```

Comment:
- do you mean "lower thrust than" (ie typo "then")?
- Can you pleae put a comma before " or velocity setpoint"

Otherwise looks great. Thanks!

Reply:
@hamishwillee, Fixed.
By The Way, I made the chnage on Github. But couldn't do the Fix here. Do you know if that possible?